tests/unit/test_api.py

```diff
@@ -0,0 +1,11 @@
+from packit_service.service.api.copr_builds import optional_time
+from datetime import datetime
+
+
+def test_optional_time():
+    time_object = datetime.utcnow()
+    nonetype_object = None
```

Comment:
Let's try to use [pytest.mark.parametrize](https://docs.pytest.org/en/latest/parametrize.html) to prepare these input values and also the expect values bellow. With that there would be 2 separate tests executed during the test run, with 2 different inputs and expectations.

This would be better, b/c you want to have your tests as atomic as possible, so that they provide you with as much information as possible, even when they fail.

Right now if the first assert fails, the test run will never get to the second assert, so you would first need to fix the first issue, before being able to tell, if the the other one works. In many cases this will slow you down.